client/src/App/reducers/users.js

```diff
@@ -13,6 +13,12 @@ const users = (state = {
                 ...state,
                 isFetching: true
             };
+        case DELETE_USER:
+            const userId = action.user._id;
+            return Object.assign({},
+                state, { users: state.users.filter(user => user._id !== userId)}
```

Comment:
do you have a recommendation of a better method for filtering? The Redux docs recommended not changing the original object if I read correctly.